h2o-core/src/main/java/water/util/Log.java

```diff
@@ -81,7 +81,7 @@ public static void httpd( String method, String uri, int status, long deltaMilli
     l.info(s);
   }
 
-  public static void info( String s, boolean stdout ) { if( _level >= INFO ) write0(INFO, stdout, s); }
+  public static void info( String s, boolean stdout ) { if( _level >= INFO ) write0(INFO, stdout, new String[]{s}); }
```

Comment:
write0( int lvl, boolean stdout, String s ) does initialize logging (it calls createLog4j). I am not sure I understand how does this fix work. Can you please explain?